app/parsers/bulkrax/application_parser.rb

```diff
@@ -134,7 +134,9 @@ def write_import_file(file)
 
     # Base path for imported and exported files
     def base_path(type = 'import')
-      ENV['HYKU_MULTITENANT'] ? File.join(Bulkrax.send("#{type}_path"), Site.instance.account.name) : Bulkrax.send("#{type}_path")
```

Comment:
this code was moved to the application parser in [this commit](https://github.com/samvera-labs/bulkrax/commit/65b2ef282aa546b29fa886c6ee23b2e84ba2b196#diff-fb85ab6a3ae748c61b9e6ca84ae5ab375b13470522978ecdd561b699967cfdf1), sept 2021.

unsure if it ever worked as intended though because `ENV['HYKU_MULTITENANT']` would have returned `'true'` or `'false'`. both of which would have evaluated to true since they're not empty strings.

Reply:
ran into the same issue in hyku, kept evaluating to true because it was actually `'false'` vs `false`